README.md

```diff
@@ -19,12 +20,14 @@ To run app you must create env files in root directory
 .development.env
 ```
 
-* **PORT** - port to run app
-* **DB_NAME** - database name
-* **DB_HOST** - database host
-* **DB_PORT** - database port
-* **DB_USERNAME** - database username
-* **DB_PASSWORD** - database password
+- **PORT** - port to run app
+- **DB_NAME** - database name
+- **DB_HOST** - database host
+- **DB_PORT** - database port
+- **DB_USERNAME** - database username
+- **DB_PASSWORD** - database password
+- **GOOGLE_CLIENT_ID** - "71848933982-oc9dn519u5k0dp3pp0a8h20maaokdap8.apps.googleusercontent.com"
+- **GOOGLE_CLIENT_SECRET** - "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"
```

Comment:
I think its secret to be a secret if you push it to github it not a secret anymore, never push any secrets to github, its very easy to check the history and get all creds. and please create .env.examples it needed for env variable, not README.md